tests/unit/__init__.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, *args, **kwargs):
         self.container = Mock(ContainerClient, perfdata=None)
         self.__dict__.update(kwargs)
         self._blob_client = Mock(BlobClient)
+        self._global_kwargs = dict()
```

Comment:
`_global_kwargs` seems unused

Reply:
`FakeStorageAPI` does not call `ObjectStorageApi` constructor, and `_global_kwargs` is not declared, but still required for some methods.